roxctl/connectivity-diff/netpol.go

```diff
@@ -84,6 +84,9 @@ func (cmd *diffNetpolCommand) construct(args []string) (diffAnalyzer, error) {
 }
 
 func (cmd *diffNetpolCommand) validate() error {
+	if cmd.inputFolderPath1 == "" || cmd.inputFolderPath2 == "" {
+		return errors.New("both directory paths dir1 and dir2 are required")
+	}
```

Comment:
handle for each argument separately, so that the msg is specific on the missing args.